src/app/supplies/page.tsx

```diff
@@ -1,69 +1,56 @@
 "use client";
+
 // ** React Imports
 import * as React from "react";
```

Comment:
In general, is a bad practice to use a wildcard to import all components from a package, it is better to be explicit. Declaring explicit components will improve maintainability of the code and potential future refactors.